lib/features/preview-support/PreviewSupport.js

```diff
@@ -20,6 +22,8 @@ import { getVisual } from '../../util/GraphicsUtil';
  * @typedef {import('../../draw/Styles').default} Styles
  */
 
+const cloneIds = new Ids('clone');

```

Comment:
```suggestion
const cloneIds = new Ids('ps');

```

Reply:
We could ensure that the ids are scoped to the component. Overlays for example uses `ov` as a prefix.

Reply:
adjusted and squashed 🚀